.tekton/tekton.yaml

```diff
@@ -0,0 +1,6 @@
+tasks:
+  - git-clone
+  - yaml-lint
```

Comment:
those are automitaclly fetch from the official catalog ?

Reply:
yep!

Reply:
all in the doc :

![image](https://user-images.githubusercontent.com/98980/97446120-5169a480-192e-11eb-80e7-20f7485a6681.png)